walkthroughs/week-4-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,15 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
-    return "Hello " + name;
+    String newName = name;
+    //Runs as long as there are spaces in the name
+    while(newName.contains(" ")){
```

Comment:
I think it's totally fine to leave this as-is since this was mostly for practice, but you might look into the `replace` and `replaceAll` functions in the `String` class!